KOALA/donations.yml

```diff
@@ -0,0 +1,129 @@
+- ticker: KOALA
+  recipient: 
+    name: Friends of the Koala inc
+    address: 23 Rifle Range Rd, East Lismore, NSW 2480, Australia
+    tel: +61266214664
+    contact: supportakoala@friendsofthekoala.org
+    
+  ada: 1164.4
+  usd: 190.38
+  timestamp: 1608173207
+  
+  files:
+    - KOALA/AdoptionCertificate_1.jpg
+    - KOALA/receipt1.PNG
+    
+  announcement_url: https://koalapool.org/pool-budget.html
+      
+- ticker: KOALA
+  recipient: 
+    name: Friends of the Koala inc
+    address: 23 Rifle Range Rd, East Lismore, NSW 2480, Australia
+    tel: +61266214664
+    contact: supportakoala@friendsofthekoala.org
+    
+  ada: 162
+  usd: 190.80
+  timestamp: 1616813207
+  
+  files:
+    - KOALA/AdoptionCertificate_2.jpg
+    - KOALA/receipt2.PNG
+    
+  announcement_url: https://koalapool.org/pool-budget.html
+  
+- ticker: KOALA
+  recipient: 
+    name: KOALA Fund Wallet
+    address: addr1qx3rdsfuq76m228lr7l402xs3d0mlnr3mfr9lt9csj9eyvcu9y9ssydecu0vesgxmpuuepcfv04sgfd3m3std3jmgwxq00txd4
```

Comment:
This is an address which is owned by you correct? When is it planned to actually donate this?
I think the USD value at the donation time will differ from the time when you move it to your fund wallet.
Also I think it is not OK to mention it as donated if it is still on a wallet under your control.

Reply:
Hey Mate, yes that's right I alone currently control the keys to that wallet. This is very far from ideal, but until Voltaire is rolled out I can't have the account controlled by delegator vote as I plan to do. I've posed in the MDP discord in regards to this, I would really like to see MDP setup some multi-sig arrangement for pools taking this path. I'll probably head that up myself in the near future as I'm very closet to ditching work and being able to spend more time on Cardano.

The pool community as a whole decided to take this approach for a few reasons. One from myself is paying $ from my pocket and keeping the ADA really felt like buying cheap ADA from Koala's and robing them of the future gain in ADA value. It felt like stealing and just a cheap way of promoting the pool. Secondly donating to Koala hospitals and sanctuaries while good is really treating the symptoms, protecting habitat is the cure. Our ultimate aim is to buy and preserve Koala (and consequently other species) habitat. The other issue is properly auditing existing Koala charities to donate to is also very hard. There are a lot of such charities with questionable practices. My delegators voted to save up for this goal of protecting habitat. The plan is to save up a fund and create a new type of charity around it. Trust in the current mainstream charities is at an all time low especially in Australia. We want to build a delegator controlled, transparent charity system. Blockchain is about replacing old financial system models, charities are apart of that which needs replacing to a degree imo.

Yes hopefully when the ADA is eventually changed to fiat the USD value will be a lot higher. Preferably it would just be a direct ADA spend in the future. However in my eyes the date at which the funds are sent to the Koala fund is the moment I donated the ADA to the cause. The USD value at the time is effectively what I have given up to the cause.

I didn't notice any other option to call it something other than a "donation". I'd like to see another option like maybe "allocation". I did have a quick chat with Brian about it some time ago but he said to just throw it in as is for now. I find this very interesting, are we choosing to only value donations/allocations to the the traditional established charities or do we actually want to also include the option of creating and developing better systems? My entire reputation rests on that KOALA fund wallet, and for the moment trust is required. But I don't see much difference, how far do we go in verifying receipts that pools post? Do we confirm these aren't forgeries, or charities with questionable accounting etc? Trust is also required there.

I have plans for building a new type of trust less fully transparent charity, but we need Voltaire to enable this. Plus time, money, people.

Reply:
Oh, I also have a 1000 ADA donation to veritree to add now. Will get to that soon.